kzg_prover/benches/kzg.rs

```diff
@@ -59,6 +72,11 @@ fn bench_kzg<const K: u32, const N_USERS: usize, const N_CURRENCIES: usize, cons
                     &advice_polys.advice_blinds,
                     &params,
                     balance_column_range,
+                    csv_total
```

Comment:
I just wonder that you want to include the polynomial evaluation inside this bench, right? if not, you can offload this computation on `setup` input instead of `routine`.

Reply:
As we discussed, it's probably negligible.